internal/config/config.go

```diff
@@ -377,6 +369,24 @@ func Default() *Config {
 			GCPSEVES:           &GCPSEVES{Measurements: measurements.DefaultsFor(cloudprovider.GCP, variant.GCPSEVES{})},
 			QEMUVTPM:           &QEMUVTPM{Measurements: measurements.DefaultsFor(cloudprovider.QEMU, variant.QEMUVTPM{})},
 		},
+		NodeGroups: map[string]NodeGroup{
+			"control_plane_default": {
+				Role:            "control-plane",
+				Zone:            "",
+				InstanceType:    "",
+				StateDiskSizeGB: 30,
+				StateDiskType:   "",
+				InitialCount:    3,
```

Comment:
why 3 and only 1 worker? I think that 1 is a sensible default that a user would change if they need to?

Reply:
3 is better since it is the minimum number that ensures high availability of the control-plane (including etcd).
My reasoning is that we want to have defaults for a sane cluster that is unlikely to need recovery.
Users will see the amount of nodes clearly displayed when running "constellation create" so they can still change this to `1` if desired.

Reply:
Curious to learn why the number is 3 :)
Can we maybe keep the node flags for us devs internally (so that we can quickly set up a test cluster with 1 node to save costs)?
(I know we can workaround with yq but seems like an unnecessary step)

I definetly think that sensible defaults for the other fields should be set. So that `config generate` + `create` works out of the box

Reply:
Here are the details of why three is a good default: https://etcd.io/docs/v3.5/faq/#deployment also https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/high-availability/#before-you-begin

> Can we maybe keep the node flags for us devs internally (so that we can quickly set up a test cluster with 1 node to save costs)?

I think as a developer you either have an existing config (with a customized number of initial nodes already set) or have to also set other config values (like debug, image). So I think we should not have a special flag and dogfood the sam UX that we want for our users. I also want to reduce the complexity of the cli.